README.md

```diff
@@ -80,6 +80,7 @@ Our e-commerce web application server, developed by Team Ninjas, facilitates smo
 | 24  | PUT    | /api/shop/seller-update-product-status/:id | 200 OK      | private | Seller update product status        |
 | 25  | GET    | /api/shop/seller-get-products              | 200 OK      | private | Seller get products                 |
 | 26  | GET    | /api/shop/user-get-products                | 200 OK      | public  | User get product                    |
+| 27  | GET    | /api/shop/buyer-getSpecific-product/:id    | 200 OK      | private | Buyer get  specific products        |
```

Comment:
I don't like the name of API 'buyer-getSpecific-product'. This is too much. Probably 'buyer-get-product' will be enough.